### ISSUE TITLE
Restarting the IDE shows the wizard again 

### ISSUE THREAD

**Issue by adben** [ChrisRM/material-theme-jetbrains#934]
After the recent update(Material Theme UI Updated to v2.9.0-pre1.) when restarting the IDE the wizard keeps showing op resetting the configuration again and again...

IntelliJ IDEA 2018.2.3 (Ultimate Edition)
Build #IU-182.4323.46, built on September 3, 2018
JRE: 1.8.0_152-release-1248-b8 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.13.6

**Comment by Clarifier** [comment]
Same here.

IntelliJ IDEA 2018.2.3 (Ultimate Edition)
Build #IU-182.4323.46, built on September 3, 2018
JRE: 1.8.0_152-release-1248-b8 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.18.5-200.fc28.x86_64

**Comment by lukoyanov** [comment]
Please fix it.

**Comment by kirkcola** [comment]
Same on Windows.

>IntelliJ IDEA 2018.2.3 (Ultimate Edition)
Build #IU-182.4323.46, built on September 4, 2018
JRE: 1.8.0_152-release-1248-b8 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

**Comment by hypercube1024** [comment]
Same on macOS 10.13.6  
IntelliJ IDEA 2018.2.3

**Comment by sho-yamane** [comment]
Please fix it.

**Comment by erikLovesMetal** [comment]
Same here on macOS High Sierra Intellij IDEA 2018.2.3

**Comment by jwenting** [comment]
More information about what's going on in my issue: https://github.com/ChrisRM/material-theme-jetbrains/issues/936

**Comment by mallowigi** [comment]
Like I said I’m taking a break and this is a alpha version. Please revert to 2.8.3

**Comment by smutek** [comment]
Merging an alpha into production, releasing it, and then taking a break seems unusual.  I manage my plugins via the IDE, the upgrade notification only tells me an update is available, it does not that mention that the update is an alpha and that the maintainer is taking a break from active development. It was only after I got frustrated that I came here to see if anyone had filed an issue.

I don't mean that to criticize you, I'm mentioning it because there's probably a lot of users experiencing the same problem who aren't going to find their way here and will just uninstall the plugin and move on. 

Reverting to 2.8.3 did the trick for me, thanks. Also and for what it's worth, in my case, the issue only impacts IntelliJ. I have not had problems with the other JetBrains IDE's I use.

**Comment by Moonpie413** [comment]
Same on Manjaro Linux
IntelliJ IDEA 2018.2.3

**Comment by Trontor** [comment]
I'm getting this same issue across two different computers, latest versions for both IntelliJ and Material Theme, it's really annoying.

**Comment by nvdlug** [comment]
Please fix this I thought I was going crazy

**Comment by smutek** [comment]
@nvdlug I downloaded it from https://plugins.jetbrains.com/plugin/8006-material-theme-ui then manually installed from disk.

**Comment by kevinwchn** [comment]
Same problem here. Every time the theme will be reset and all the settings are gone. Please fix it

**Comment by mallowigi** [comment]
Duplicate of #936
